src/static/css/module/loan-comparison.less

```diff
@@ -364,63 +364,139 @@
   --------------------------- */
 
 .comparison-results {
-  margin-top: 60px;
+  .expandable {
+    .respond-to-min(@tablet-min, {
+      overflow: hidden;
+    });
+  }
+  .expandable_header {
+    margin-bottom: 80px;
+    background: @gray-10;
+  }
+  .expandable_header-left {
+    // .respond-to-min(@tablet-min, {
+    //   width: 90%;
+    // });
+  }
+  .expandable__padded,
+  .expandable__padded.expandable__expanded {
+    background: @white;
+  }
+  .expandable__padded .expandable_content {
+    margin-right: 0;
+    margin-left: 0;
+    background: @white;
+    &:before,
+    &:hover {
+      display: none;      
+    }
+  }
 }
 
 .result-heading {
-  color: @gray;
   .grid_push(2);
 }
 
 .results-section {
+  position: relative;
   margin-bottom: 60px;
 }
+  .results-section-heading {
+    margin: 0;
+    .cf-icon {
+      color: @navy;
+    }
+    .respond-to-min(@tablet-min, {
+      font-size: 1.375em;
+    });
+  }
 
-.results-section-heading {
-  .reset;
-  .webfont-medium();
-  font-size: 1.125em;
-  background-color: @gray-10;
-}
+  .primary-results-row {
+    position: absolute;
+    top: 55px;
+    clear: both;
+    .respond-to-min(@tablet-min, {
+      float: left;
+      clear: none;
+    });
+  }
 
```

Comment:
This is a relic of when I had `.primary-results-row` inside the button container, I can remove this.